integrations/tensorflow-keras/setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup
+
+
+def main():
+    with open('README.md') as readme_file:
+        readme = readme_file.read()
+
+    extras = {
+    }
+
+    all_deps = []
+    for group_name in extras:
+        all_deps += extras[group_name]
+    extras['all'] = all_deps
+
+    base_libs = ['neptune-client>=0.5.1']
```

Comment:
I don't think this will be needed in this package - get rid of those lines?

Reply:
we will change this to minimal supported client version after i gets released